### PR TITLE
Acceptance email support

### PR DESCRIPTION
**Pull request by akruj17**

### CODE REVIEW

**Comment by akruj17** [comment]
acceptance email with the callback issue fixed

File: app/server/controllers/UserController.js [hunk 1]
```diff
@@ -538,6 +538,24 @@ UserController.sendPasswordResetEmail = function(email, callback){
...
+ */
+UserController.sendAcceptanceEmail = function(email, callback){
+  User
+    .findOneByEmail(email)
+    .exec(function(err, user){
+      if (err || !user){
+        return callback(err);
+      }
+
+      Mailer.sendAcceptanceEmail(email, callback);
```
Comment (copperstick6) [inline app/server/controllers/UserController.js]: A parameter of this function is a callback, but you're not passing in a callback. Look above for more details.

File: app/client/views/admin/users/adminUsersCtrl.js [hunk 2]
```diff
@@ -117,12 +118,13 @@ angular.module('reg')
...
 
-                UserService
+               UserService
                   .admitUser(user._id)
-                  .success(function(user){
+                  .success(function(user) {
                     $scope.users[index] = user;
                     swal("Accepted", user.profile.name + ' has been admitted.', "success");
-                  });
+                    AuthService.sendAcceptanceEmail(user.email);
```
Comment (copperstick6) [inline app/client/views/admin/users/adminUsersCtrl.js]: One of the parameters of the sendAcceptanceEmail is a callback. If that method fails, we should know about it (the method should fail gracefully). I know this is a bit nitpicky, but unexpected issues could come up in production, and we'll spend more time debugging the crash. This way we'll know exactly what went wrong. I'd recommend adding a function that logs it somewhere.

Comment (shreyassood) [reply app/client/views/admin/users/adminUsersCtrl.js]: Also, I think the email acceptance method doesn't necessarily need to live in AuthService, you can modify it just to live in the UserService where it's invoked in admitUser automatically or through another method in UserService.

Comment (yursky) [reply app/client/views/admin/users/adminUsersCtrl.js]: Keeping it in UserService stays consistent with the rest of the app layout. AuthService should keep its scope where it is without adding it in other places.

**Pull request merged**